src/ui/public/field_editor/field_editor.js

```diff
@@ -450,17 +450,10 @@ export class FieldEditorComponent extends PureComponent {
         >
           <p>
             <FormattedMessage
-              id="common.ui.fieldEditor.deleteField.label.detail"
-              defaultMessage="You can't recover a deleted field. {question}"
+              id="common.ui.fieldEditor.deleteField.label"
+              defaultMessage="You can't recover a deleted field.{separator}Are you sure you want to do this?"
```

Comment:
I guess this is OK though I would still prefer to have two paragraphs. Perhaps we could label them in a way that makes it clear that they're related? e.g. `common.ui.fieldEditor.deleteField.label.p1`, `common.ui.fieldEditor.deleteField.label.p2`?

There are parts of our UI where we have many paragraphs. It seems like it would be difficult to maintain them if we have to put them all in a single string like this.

Reply:
We could, but I would personally suggest to use a progressive number in the key only as a last resort.